.travis.yml

```diff
@@ -1,40 +1,22 @@
 # Config file for automatic testing at travis-ci.org
 language: python
+dist: xenial
 python:
   - 2.7
   - 3.4
   - 3.5
   - 3.6
-
-matrix:
-  include:
-    - python: 3.7
-      dist: xenial
-      sudo: true
+  - 3.7
```

Comment:
why this change? to the best of my understanding, the previous state was the correct way to test python 3.7: https://github.com/travis-ci/travis-ci/issues/9815. the way you have done it seems like it should not work because sudo is required. and removing the matrix include entry means that you would have to enable sudo for all environments which is less than ideal.

Reply:
The change is because this configuration is simpler and it is proved to work.

Reply:
where? the best reference I have seen is the linked travis issue.

Reply:
Originally I got it from here: https://github.com/travis-ci/travis-ci/issues/9831
Later on I made it work here: https://github.com/csala/smapy/blob/master/.travis.yml
Also the travis python guide points at a similar approach: https://docs.travis-ci.com/user/languages/python/

Reply:
fair enough